<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1111</width>
    <height>861</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Random">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>740</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(255, 255, 255);
}

QPushButton:hover{
	background-color: rgb(41, 205, 99);
};</string>
    </property>
    <property name="text">
     <string>Oponente</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Finish">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>740</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(255, 255, 255);
}

QPushButton:hover{
	background-color: rgb(41, 205, 99);
};</string>
    </property>
    <property name="text">
     <string>Reiniciar</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>1011</width>
      <height>691</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="OpenGLLayout">
     <property name="spacing">
      <number>4</number>
     </property>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1111</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuJuego">
    <property name="title">
     <string>Juego</string>
    </property>
    <addaction name="actionJugadas"/>
    <addaction name="actionCerrar"/>
   </widget>
   <addaction name="menuJuego"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCerrar">
   <property name="text">
    <string>Cerrar</string>
   </property>
  </action>
  <action name="actionReiniciar">
   <property name="text">
    <string>Reiniciar</string>
   </property>
  </action>
  <action name="actionJugadas">
   <property name="text">
    <string>Total Jugadas</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
